### PR TITLE
Publish @pxblue/angular-components v4.1.0

### PR DESCRIPTION
**Pull request by emclaug2**
#### Changes proposed in this Pull Request:
- Release @pxblue/angular-components@4.1.0 

### Added

-   Adds new property `openOnHover` to `<pxb-drawer>`.
-   Adds new class `pxb-dropdown-toolbar-subtitle-icon` to `<pxb-dropdown-toolbar>`
-   Adds lighter font weight to `<pxb-info-list-item>` placed inside `<pxb-user-menu>`.
-   Adds a shadow to the `<pxb-user-menu>` when opened.
-   Adds new property `openOnHoverDelay` to `<pxb-drawer>` to alter open-on-hover delay for closed persistent drawers. 

### Fixed

-   Fixes bug in `<pxb-mobile-stepper>` which makes component span 100% of parent width.
-   Fixes header height bug which affected `<pxb-drawer-header>` in Safari.
-   Fixes misalignment of `<pxb-info-list-item>`'s right content.
-   Fixes RTL styles in `<pxb-dropdown-toolbar>`.
-   Fixes uneven vertical alignment of icon and text in `<pxb-empty-state>` action buttons. 
-   Fixes `<pxb-user-menu>` menu header avatar offset.
-   Fixes `<pxb-drawer>` `sideBorder` prop to use `mat-elevation` class.  

### Removed

-   Removes automatic RTL flip logic from user-provided icons to `<pxb-info-list-item>`.

### Changed

-   Default styles of `<pxb-dropdown-toolbar>` for more padding around the dropdown arrow.
-   Changed `<pxb-user-menu>` menu title default font size to 16px.
-   Changed `<pxb-user-menu>` menu subtitle default font size to 14px.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,36 +1,85 @@
 # Change Log
 
+## v4.1.0 (Not Published Yet)
```
Comment (joebochill) [inline CHANGELOG.md:3]: You need to take out the "Not Published Yet" since this will publish it

**Review by joebochill — APPROVED** [review]

**Comment by jeffvg** [comment]
Is there documentation changes needed for rail condensed and hover tooltips?

**Comment by emclaug2** [comment]
> Is there documentation changes needed for rail condensed and hover tooltips?

The rail/hover tooltip was something that was introduced in a previous release; we have some docs for it.  There was one drawer change that's included in this released, the `openOnHover ` was added and we have some docs around it.

**Comment by jeffvg** [comment]
is it possible to get some padding in here for this label "100"? Everything else looks good

image

**Review by jeffvg — APPROVED** [review]

**Pull request merged**